src/api/api.ts

```diff
@@ -0,0 +1,13 @@
+const GET_MOVIE_API = 'https://www.omdbapi.com/?apikey=dc7441d9&t=';
+
+export const getMovieByTitle = (title: string): Promise<Movie> => {
+  return fetch(`${GET_MOVIE_API}${title}`)
+    .then(responce => responce.json())
```

Comment:
try to use async/await (everywhere), it's more common in real projects

Reply:
done!